src/Cms/Auth.php

```diff
@@ -221,7 +221,7 @@ public function login(string $email, string $password, bool $long = false)
         ];
 
         // validate the user and log in to the session
-        $user = $this->validatePassword($email, $password);
+        $user = $this->validatePassword(User::decodeEmail($email), $password);
```

Comment:
I think the conversion should be moved to the `validatePassword()` method so that it applies to all ways of using the class.

The `validatePassword()` method then also needs a test for both cases (input is a Unicode string or a Punycode string).